### PR TITLE
Upgraded ptb to the latest version , chatched akitimed out issue 

### PR DESCRIPTION
**Pull request by blacknut1803**

### CODE REVIEW

**Comment by advnpzn** [comment]
Hi @nooneofyourbuisness , first of all, thanks for trying to improve Akinator Bot!

So, I have noticed that you have updated the `python-telegram-bot` version from **13.x** to **20.x**
This is a major version bump and there could be incompatible functions/methods used in Akinator Bot that may not have followed the new practices set forth upon by PTB. So, I wanted to know if all the functions are working as expected and there are no errors, Can you confirm that everything is working fine based on the changes you have made ?

One other thing is the second part of your title after the comma.
**"chatched akitimed out issue"**

I'm not sure if I can understand what you're trying to say here.
Looking forward to your reply.

Thanks,
mito/adenosinetp10

File: database.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 from pprint import pprint
```
Comment (advnpzn) [inline database.py]: I think we can remove `pprint` import as well since it is not being used anywhere in the code.

File: __main__.py
Comment (advnpzn) [inline __main__.py:1]: You have rearranged the functions quiet a lot so it's hard for me to follow what you have changed with respect to what was already there.

The code is quiet old and I may not remember a lot of things I did at face value. So I may need some time to look at this thoroughly.

**Comment by blacknut1803** [comment]
> Hi @nooneofyourbuisness , first of all, thanks for trying to improve Akinator Bot!
> 
> So, I have noticed that you have updated the `python-telegram-bot` version from **13.x** to **20.x** This is a major version bump and there could be incompatible functions/methods used in Akinator Bot that may not have followed the new practices set forth upon by PTB. So, I wanted to know if all the functions are working as expected and there are no errors, Can you confirm that everything is working fine based on the changes you have made ?
> 
> One other thing is the second part of your title after the comma. **"chatched akitimed out issue"**
> 
> I'm not sure if I can understand what you're trying to say here. Looking forward to your reply.
> 
> Thanks, mito/adenosinetp10

yes I have checked with my TOKEN and MONGO_DB it is working fine 
sorry about the spelling mistakes i meant to say caught the exception **akinator.exceptions.AkiTimedOut** 
here is the [documentation](https://akinator.readthedocs.io/en/latest/#:~:text=%2C%20and%20AkiTechnicalError-,AkiTimedOut,Raised%20if%20the%20Akinator%20session%20times%20out.%20Derived%20from%20AkiConnectionFailure,-AkiNoQuestions)
previously  If the user clicks the button very late after starting the game, the don't responds .

File: database.py (hunk 1 above)
Comment (advnpzn) [reply database.py]: @nooneofyourbuisness Could you please remove this line ?
```python
from pprint import pprint
```

File: __main__.py
Comment (advnpzn) [reply __main__.py:1]: Looks fine to me.

File: database.py (hunk 1 above)
Comment (advnpzn) [reply database.py]: Just remove this line and I think we are good to merge.

**Review by advnpzn — CHANGES_REQUESTED** [review]
Remove the `pprint` import in `database.py` file since we are not using it.

File: database.py (hunk 1 above)
Comment (blacknut1803) [reply database.py]: done removed `from pprint import pprint` also added try except catch on line 169-172 which i forgot previously in __main__.py

**Review by advnpzn — APPROVED** [review]

**Comment by advnpzn** [comment]
Thank you @nooneofyourbuisness

I've merged your request.

**Pull request merged**